speed up tests by 20 seconds

dannyroberts opened the pull request:
by settings `SOUTH_TESTS_MIGRATE = False`

@czue do you remember why we didn't have this set?

Before this the overhead of just running a single db test is 30+ seconds of setup; with this it's down to ~11.

dannyroberts committed:
speed up tests by 20 seconds

by settings SOUTH_TESTS_MIGRATE = False

dimagimon commented on settings.py:
```diff
@@ -941,9 +939,9 @@
-####### South Settings #######
-#SKIP_SOUTH_TESTS=True
-#SOUTH_TESTS_MIGRATE=False
+####### South Settings ###### #
```
E265 block comment should start with '# '

czue commented:
I think the main reason is that it prevented us from detecting bugs in our migrations. I think we turned it back on after that happened once. Seems like the 20 seconds is worth it on travis and we should just encourage devs to turn it off locally.

dannyroberts committed:
make travis run SOUTH_TESTS_MIGRATE = True

dimagimon commented on settings.py:
```diff
@@ -891,6 +889,11 @@
 # override for production
 DEFAULT_PROTOCOL = 'http'
 
+####### South Settings #######
```
E265 block comment should start with '# '

dannyroberts commented:
Ok, so I moved `SOUTH_TESTS_MIGRATE = False` to above the localsettings import, and added `SOUTH_TESTS_MIGRATE = True` to .travis/localsettings.py. I think that's basically what you suggested, but with less hassle remembering to tell it to new devs, etc. People can still run revert to using the more conservative behavior by adding `SOUTH_TESTS_MIGRATE = True` to their localsettings. Sound good?

benrudolph commented:
+1 to using SOUTH_TESTS_MIGRATE. saved me loads of time on tests

czue commented:
yep this looks perfect. i'll wait for build just for kicks.
